python/dgl/contrib/cugraph/cugraph_utils.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph
+import cudf
+import dgl
+import cupy
+
+
+# from cugraph to DGL using
+def cugraphToDGL(graph):
```

Comment:
Let's have a separate PR to add this function to DGL. You could put the function in `dgl/convert.py`. Suggested signature:
```python
def from_cugraph(cug):
    """Create a graph from a cuGraph graph object.

    Parameters
    ----------
    cug : cugraph.Graph
        The input cuGraph object.

    Returns
    -------
    g_dgl : DGLGraph
        The created DGL graph object.
    """
```

Reply:
Thanks for the review, Will create a separate PR for this.

Reply:
Started a PR for this https://github.com/dmlc/dgl/pull/4166 here.